packages/features/users/components/TwoFactorAuthRequiredBanner.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSession } from "next-auth/react";
+
+import { APP_NAME } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { TopBanner } from "@calcom/ui";
+import { Key } from "@calcom/ui/components/icon";
+
+function TwoFactorAuthRequiredBanner() {
```

Comment:
Banner to prompt users that they are required to setup mfa.
This helps avoid confusion since they will not be able to navigate
to other pages until they setup 2fa.